debug/debugging.rst

```diff
@@ -56,5 +56,57 @@ by removing the call to ``loadClassCache()``::
     session.
 
 Some IDEs do not like the fact that some classes are stored in different
-locations. To avoid problems, you can tell your IDE to ignore the PHP cache
-file.
+locations. To avoid problems, you can either tell your IDE to ignore the PHP
+cache files, or you can change the extension used by Symfony for these files::
+
+    $kernel->loadClassCache('classes', '.php.cache');
+
+Useful debugging commands
+-------------------------
+
+When developing a large application, it can be hard to keep track of all the
+different services, routes and translations. Luckily, Symfony has some commands
+that can help you visualize and find the information.
+
+Information about the contents of the Symfony container can be found using the
+command:
+
+.. code-block:: terminal
```

Comment:
You have to use `terminal` instead of `bash`. This is a custom lexer we use for terminal examples.

However, we no longer maintain the 3.0 branch, which is why it doesn't exists there. If possible, please rebase this branch against 2.7 (the oldest still maintained branch) using these commands:
```
# assumed origin is symfony/symfony-docs and hiddewie is your fork
$ git fetch origin
$ git checkout debugger-commands2
$ git rebase --onto origin/2.7 origin/3.0
$ git push hiddewie --force debugger-commands2
```

Then, click the "edit" button of this PR and also change the base branch to 2.7.